CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Bug Fixes
 
 - Fixed issue with Go dependencies caused by removed tag in `openshift/api` repository ([#2466](https://github.com/operator-framework/operator-sdk/issues/2466))
+- Fixed a regression in the `operator-sdk run` command that caused `--local` flags to be ignored (#2478)
```

Comment:
```suggestion
- Fixed a regression in the `operator-sdk run` command that caused `--local` flags to be ignored ([#2478](https://github.com/operator-framework/operator-sdk/issues/2478))
```

Reply:
Sorry, my bad. I've missed that. Fixed.

Reply:
No worries. I should have been more clear the first time :)